tests.json

```
@@ -0,0 +1,243 @@
+{
+    "TestA": {
+      "name": "Build_Test",
+      "test": "BuildTest",
+      "var1": "/opt/Validation/build.sh",
+      "var2": "/opt/WCSim/install/bin/WCSim",
+      "comment": "Tests defined by suffixed letters were commented out in original tests.txt, they are kept here for comepleteness :)"
+    },
+    "TestB": {
+      "name": "100e_SuperK",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/100e_SuperK.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_100e_SuperK.root"
+    },
+    "TestC": {
+      "name": "10e_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/10e_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_10e_HyperK20pc.root"
+    },
+    "TestD": {
+      "name": "100e_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/100e_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_100e_HyperK20pc.root"
+    },
+    "TestE": {
+      "name": "500e_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/500e_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_500e_HyperK20pc.root"
+    },
+    "TestF": {
+      "name": "200mu_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/200mu_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_200mu_HyperK20pc.root"
+    },
+    "TestG": {
+      "name": "800mu_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/800mu_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_800mu_HyperK20pc.root"
+    },
+    "TestH": {
+      "name": "100pi0_HyperK20pc",
+      "test": "PhysicsValidation",
+      "var1": "/opt/Validation/Generate/Generate.sh",
+      "var2": "/opt/Validation/Generate/macReference/100pi0_HyperK20pc.mac",
+      "var3": "/opt/Validation/Compare/Reference/analysed_100pi0_HyperK20pc.root"
+    },
+    "Test1": {
+      "name": "TEST_SK_5MeVem_centre_xdir",
+      "test": "PhysicsValidation",

```

```suggestion
      "test": "WCSimPhysicsValidation",
```
Might be nice to be clearer here?
Then in the `RunTests.py` can do something like 
```py
if test == f"{SOFTWARE}PhysicsValidation":
```

---

Can do this, making SOFTWARE another environment variable maybe?

Or possibly an argument for the script?

---

Yeah. Or a config file variable. Not sure which is best out of the 3

(of course `SOFTWARE` is a bad name, particularly for an environment variable. Something like `VALIDATION_SOFTWARE` could be better)

---

I've left it as an environment variable for now, as I think it can be added with a bigger change you suggested in having another config file (https://github.com/WCSim/Validation/pull/37#discussion_r1524683990)